typed_floats/src/macros.rs

```diff
@@ -179,3 +179,63 @@ macro_rules! assert_float_eq {
         }
     }};
 }
+
+/// Macro to create a constant with a positive value.
+/// The resulting constant will be of type `StrictlyPositiveFinite`.
+/// Will panic at compile time if the value is not a valid.
+///
+/// # Examples
+///
+/// ```
+/// # use typed_floats::*;
+/// const X: StrictlyPositiveFinite = positive_const!(1.0); // f64
+/// const Y: StrictlyPositiveFinite<f64> = positive_const!(f64, 2.0);
```

Comment:
```suggestion
/// const two: StrictlyPositiveFinite<f64> = positive_const!(2.0, f64);
```

I would flip the order.  

In core rust, you would do this: `2.0f64`, 
so, using the same order is probably more intuitive for rust experts

Reply:
Evidence: https://doc.rust-lang.org/rust-by-example/types/literals.html